Jenkinsfile

```diff
@@ -51,7 +53,7 @@ matrices = [
     "CYTHON": DEFAULT_CYTHON
   ],
   "SMOKE": [
-    "SERVER": DEFAULT_CASSANDRA.takeRight(2) + DEFAULT_DSE.takeRight(1),
+    "SERVER": DEFAULT_CASSANDRA.takeRight(2) + DEFAULT_DSE.takeRight(2) + DEFAULT_HCD.takeRight(1),
```

Comment:
For the record: I had this wrong in my earlier comment.  We should actually be adding only four new builds to our smoke test build matrix with this change: 2 new backends * 2 Python versions (oldest and newest).  That should be reasonably manageable.  More to the point it's hard to imagine how we'd remove either 6.8.x, 6.9.x or HCD from our test matrix.